packages/feedback-spa/src/app/feedback/dropdown/dropdown.component.ts

```diff
@@ -106,19 +106,21 @@ export class DropdownComponent implements OnInit {
     }
   }
 
-  // tslint:disable-next-line:use-life-cycle-interface
+  // tslint:disable-next-line: use-lifecycle-interface
```

Comment:
Do not disable a tslint rule. You can fix this by implementing the AfterViewInit in your class definition.
Always check the reason for a lint error/warning before disabling the rule.